src/MagnumPlugins/AnyImageConverter/AnyImageConverter.h

```diff
@@ -68,18 +68,18 @@ Supported formats for uncompressed data:
 
 -   Basis Universal (`*.basis`), converted with any plugin that provides
     `BasisImageConverter`
+-   JPEG (`*.jpg`, `*.jpe`, `*.jpeg`), converted with @ref JpegImageConverter
+    or any other plugin that provides it
 -   OpenEXR (`*.exr`), converted with any plugin that provides
     `OpenExrImageConverter`
--   Windows Bitmap (`*.bmp`), converted with any plugin that provides
-    `BmpImageConverter`
 -   Radiance HDR (`*.hdr`), converted with any plugin that provides
     `HdrImageConverter`
--   JPEG (`*.jpg`, `*.jpe`, `*.jpeg`), converted with @ref JpegImageConverter
-    or any other plugin that provides it
 -   Portable Network Graphics (`*.png`), converted with @ref PngImageConverter
     or any other plugin that provides it
 -   Truevision TGA (`*.tga`, `*.vda`, `*.icb`, `*.vst`), converted with
     @ref TgaImageConverter or any other plugin that provides it
+-   Windows Bitmap (`*.bmp`), converted with any plugin that provides
+    `BmpImageConverter`
```

Comment:
Um, the ordering should be by the extension, not by the "marketing name", as that's what people look for. It was mostly okay except for bmp :)